input/witnessgen.go

```diff
@@ -35,11 +35,32 @@ type WitnessType interface {
 	SizeUpperBound() (int, bool, error)
 
 	// AddWeightEstimation adds the estimated size of the witness in bytes
-	// to the given weight estimator and returns the number of
-	// CSVs/CLTVs used by the script.
-	AddWeightEstimation(estimator *TxWeightEstimator) (int, int, error)
+	// to the given weight estimator.
+	AddWeightEstimation(estimator *TxWeightEstimator) error
+
+	// TimeLockType returns the type of time lock used by a script of this
+	// witness type (if any).
+	TimeLockType() TimeLockType
```

Comment:
Does this mean that the interface cannot be extended to types having both CSV and CLTV?

Reply:
Oh, these are coming for anchors, isn't it?

Reply:
At the moment, yes. I think we can extend this if we need it. And since the witness only spends one path of an output it is unlikely that there would be a CSV and CLTV in the same script path. At least for the use cases of lnd and loop this should be sufficient. But I can add a type `Both` if you think that could be used in the future.

Reply:
Looking at the usage of `getWeightEstimate` it seems that the num,ber of csv/cltv is only used for logging. Should we do ourselves a favor and just remove it entirely ?

Reply:
Yeah, that's the other possibility... I'm not sure how important that log statement there really is. So I didn't just want to remove stuff. But it would remove quite a bit of code.

Reply:
Or just make a nice helper method that shows how many of each witness type we have in the sweep tx? I would like that. Also good for anchor out dev.

Reply:
sgtm

Reply:
Ok, I changed it according to @joostjager's suggestion.